### PR TITLE
Rabitmq - update to version 3.8.18

### PR DESCRIPTION
**Pull request by EugeneKorolevich**

### CODE REVIEW

File: versions.yaml [hunk 1]
```diff
@@ -21,43 +21,25 @@
 cloudbuild:
   enable_parallel: true
 versions:
+
```
Comment (ovk6) [inline versions.yaml:24]: remove newline

File: versions.yaml [hunk 2]
```diff
@@ -21,43 +21,25 @@
...
 - dir: 3/debian10/3.8
   from: marketplace.gcr.io/google/debian10
   templateSubDir: default
```
Comment (ovk6) [inline versions.yaml:27]: - remove `templates/3.7` directory as it is not used anymore
- move contents of `templates/default/*` to `templates/` and remove `templateSubDir: default`

**Review by ovk6 — CHANGES_REQUESTED** [review]

File: versions.yaml (hunk 2 above)
Comment (EugeneKorolevich) [reply versions.yaml:27]: done

File: versions.yaml (hunk 1 above)
Comment (EugeneKorolevich) [reply versions.yaml:24]: done

**Review by armandomiani — APPROVED** [review]

**Review by ovk6 — APPROVED** [review]

**Pull request merged**